src/js/components/tour/styles.tsx

```diff
@@ -0,0 +1,53 @@
+import { Styles } from 'react-joyride';
+
+import popoverHeaderImg from '~img/popover-header.png';
+
+const tourStyles: Styles = {
+  options: {
+    arrowColor: '#032e61',
+    backgroundColor: '#032e61',
+    textColor: '#ffffff',
+    primaryColor: '#0070d2',
+  },
+  buttonBack: {
+    border: '1px solid #dddbda',
+    borderRadius: '.25rem',
+    color: '#ecebea',
+    fontSize: '13px',
```

Comment:
Do we need to use px values for fonts? Worried this won't respond well to font preferences.